scenes/location.ts

```diff
@@ -11,7 +11,10 @@ import { IPrayTime, IUser } from '#types/database'
 let scene = new Scene<BotContext>('Location')
 
 scene.step(async (ctx) => {
-  const message = HLanguage('chooseRegion')
+  let currentRegionMsg = HLanguage('currentRegion');
+  currentRegionMsg = HReplace(currentRegionMsg, ['$region'], [ctx.user.region])
```

Comment:
would u agree if just added current location after msg instead of replacing

Reply:
for sure